doc/sphinx/source/changelog.rst

```diff
@@ -4,6 +4,103 @@ Changelog
 =========
 
 
+.. _changelog-v2-7-0:
+
+v2.7.0
+------
+
+Highlights
+~~~~~~~~~~
+
+-  This release has seen the inclusion of the code for figures 3.3, 3.4, 3.5, 3,13 and 3.15 of the IPCC AR6 WG1 report, see them in the `new documentation <https://esmvaltool--2533.org.readthedocs.build/en/2533/recipes/recipe_ipccwg1ar6ch3.html>`__
+-  We have also included new diagnostics and recipe necessary to produce the plots and tables for the journal article "Climate model projections from the Scenario Model Intercomparison Project (ScenarioMIP) of CMIP6" by `Tebaldi et al. <ESD. https://doi.org/10.5194/esd-2020-68>`__ from 2021
```

Comment:
I cannot open this link. Also, maybe this entry should also point to the documentation for the recipe?

Reply:
good points, Saskia, let me fix both

Reply:
fixed them proper now, cheers for spotting it!